packages/ui5-proxy-middleware/src/ui5/middleware.ts

```diff
@@ -86,6 +87,12 @@ module.exports = async ({ resources, options }: MiddlewareParameters<UI5ProxyCon
         logger.debug(error);
     }
 
+    if (await isUI5VersionRemoved(ui5Version)) {
+        throw new Error(
+            `You are trying to load the application preview with the non existing version ${ui5Version} of SAPUI5. Please check and correct the version.`
+        );
+    }
+
```

Comment:
Please don't do any changes to the `ui5-proxy-middleware`. Any specific UI5 version handling needs to be done in a package that integrates the `ui5-proxy-middleware`. OR in this case either in `adp-tooling` or in the `preview-middleware-client`.

Reply:
Removed as discussed.